python/tank/util/shotgun/download.py

```diff
@@ -177,8 +176,6 @@ def download_and_unpack_attachment(sg, attachment_id, target, retries=5):
             broadband_speed_bps = file_size * 8.0 / time_to_download
             broadband_speed_mibps = broadband_speed_bps / (1024 * 1024)
             log.debug("Download speed: %4f Mbit/s" % broadband_speed_mibps)
-            log_user_attribute_metric("Tk attachment download speed", "%4f Mbit/s" % broadband_speed_mibps)
```

Comment:
rather than just deleting it, can this be turned into a metric which works with the new system?

Reply:
@robblau what's your preference here?

Reply:
I think I recall a discussion, one of the conclusion is that from part of the code, we don't have access to engine and relevant information. If we were to log an event metric for this particular case, it might be hard to 'connect' it to the context it pertains to.

Reply:
This was the metric that let to my comment quoted above.  It doesn't cleanly fit in and if we were to design out the tracking of downloading bundles I'm guessing we'd do it at a higher level where we'd get more context about the event than just the download speed.  I did check with Ben about using Amplitude for performance metrics and while we're not doing that yet he didn't think it was something we should avoid.

For simplicity and for keeping the event stream clean as we get started with Amplitude I opted out of this particular metric, but still want to see a list of what else is unspec'ed (like the bootstrap call mentioned above) to see if we need to add a new event type to our spreadsheet.

Reply:
The only immediate one I came accros would be a boolean added by Jean-François Boismenu in Desktop: https://github.com/shotgunsoftware/tk-desktop/blob/master/python/tk_desktop/desktop_engine_site_implementation.py#L315

Reply:
That boolean feels like something we can leave out.  Just starting the websocket server isn't overly interesting for Analytics, using it for a particular purpose potentially is, but we'd need to design that out.

Looks good.

Reply:
it is not possible to retrieve the metrics _context_ here, because the `current_bundle` is unknown, so this should be moved at a higher level if still needed.